src/test/java/uk/gov/pay/api/resources/error/ApiErrorResponseTest.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.pay.api.resources.error;
+
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class ApiErrorResponseTest {
```

Comment:
This test is fine but i think it should be tested on an integration test as the user need is that the calling client will be shown the error message?

Reply:
we may already have integration tests for 429 errors : https://github.com/alphagov/pay-publicapi/blob/pp_4073_429_response_for_endpoints/src/test/java/uk/gov/pay/api/it/ResourcesFilterRateLimiterITest.java#L36

Reply:
ok. then i'm not so sure why we need this unit test as it feels like an extra unit test for the sake of it. this is just my opinion though. it'd be good to get a consensus from the rest of the dev team as to whether we need double testing at integration and unit level.

Reply:
This is added inline with https://github.com/alphagov/pay-publicapi/blob/pp_4073_429_response_for_endpoints/src/test/java/uk/gov/pay/api/model/PaymentErrorTest.java we had in place.

In this case I see your point that unit test is not adding much value as toString() is never used. So removed this test.